src/main/java/com/msalvatore/graphqlplayground/scalar/datetime/GraphQLLocalDate.java

```diff
@@ -0,0 +1,73 @@
+package com.msalvatore.graphqlplayground.scalar.datetime;
+
+import graphql.language.StringValue;
+import graphql.schema.Coercing;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.CoercingSerializeException;
+import graphql.schema.GraphQLScalarType;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+
+// TODO: This could probably be simplified/re-written, but it's a proof of concept ¯\_(ツ)_/¯
+// Taken from https://github.com/donbeave/graphql-java-datetime
+public class GraphQLLocalDate extends GraphQLScalarType {
```

Comment:
This is a key point here. The class must extend `GraphQLScalarType`, and the name passed to the constructor is the name that you'll use in the schema.